<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>439</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Source Code Pro</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>HITers</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/images.qrc">
    <normaloff>:/HITers.ico</normaloff>:/HITers.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonNews">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>工大动态</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGrade">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>成绩查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClass">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>课表查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>-10</y>
      <width>571</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/HIT.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res/images.qrc"/>
 </resources>
 <connections/>
</ui>
